resource_control: enhance observability (#15934)
pingcap/docs: statement-summary-tables.md

> **Note:**
>
> - In TiDB, the time unit of fields in statement summary tables is nanosecond (ns), whereas in MySQL the time unit is picosecond (ps).

> > - Starting from v7.6.0, for clusters with [resource control](/tidb-resource-control.md) enabled, `statements_summary` will be aggregated by resource group, i.e., the same statements executed in different resource groups will be collected as different records.

Reviewer: ```suggestion
> - Starting from v7.6.0, for clusters with [resource control](/tidb-resource-control.md) enabled, `statements_summary` will be aggregated by resource group, for example, the same statements executed in different resource groups will be collected as different records.
```

- `AVG_AFFECTED_ROWS`: The average number of rows affected.
- `PREV_SAMPLE_TEXT`: When the current SQL statement is `COMMIT`, `PREV_SAMPLE_TEXT` is the previous statement to `COMMIT`. In this case, SQL statements are grouped by the digest and `prev_sample_text`. This means that `COMMIT` statements with different `prev_sample_text` are grouped to different rows. When the current SQL statement is not `COMMIT`, the `PREV_SAMPLE_TEXT` field is an empty string.

Fields related to Resource Control:

> - `Max_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.

Reviewer 2: ```suggestion
- `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
- `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
```

Reviewer 2: missing a field, also added in docs-cn

- `AVG_AFFECTED_ROWS`: The average number of rows affected.
- `PREV_SAMPLE_TEXT`: When the current SQL statement is `COMMIT`, `PREV_SAMPLE_TEXT` is the previous statement to `COMMIT`. In this case, SQL statements are grouped by the digest and `prev_sample_text`. This means that `COMMIT` statements with different `prev_sample_text` are grouped to different rows. When the current SQL statement is not `COMMIT`, the `PREV_SAMPLE_TEXT` field is an empty string.

Fields related to Resource Control:

> - `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
> - `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
> - `AVG_REQUEST_UNIT_WRITE`: the average number of write RUs consumed by SQL statements.
> - `MAX_REQUEST_UNIT_WRITE`: the maximum number of write RUs consumed by SQL statements.
> - `AVG_REQUEST_UNIT_READ`: the average number of read RUs consumed by SQL statements.
> - `MAX_REQUEST_UNIT_READ`: the maximum number of read RUs consumed by SQL statements.
> - `RESOURCE_GROUP`: the resource group bound to SQL statements.

Author: ```suggestion
- `AVG_REQUEST_UNIT_WRITE`: the average number of write RUs consumed by SQL statements.
- `MAX_REQUEST_UNIT_WRITE`: the maximum number of write RUs consumed by SQL statements.
- `AVG_REQUEST_UNIT_READ`: the average number of read RUs consumed by SQL statements.
- `MAX_REQUEST_UNIT_READ`: the maximum number of read RUs consumed by SQL statements.
- `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
- `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
- `RESOURCE_GROUP`: the resource group bound to SQL statements.
```
